Observer/AfterConfigSaveObserver.php

```diff
@@ -87,19 +89,30 @@ public function execute(Observer $observer)
             {
                 $webhookPresent = $this->getExistingWebhook();
 
-                if(empty($razorpayParams['enable_webhook']['value']) === true)
-                {
-                    $this->disableWebhook();
-                    return;
-                }
-
                 $events = [];
 
                 foreach($razorpayParams['webhook_events']['value'] as $event)
                 {
                     $events[$event] = true;
                 }
 
+                if(empty($this->config->getConfigData('webhook_secret')) === false)
+                {
+                    $razorpayParams['webhook_secret']['value'] = $this->config->getConfigData('webhook_secret');
+
+                    $this->logger->info("Razorpay Webhook with existing secret.");
+                }
+                else
+                {
+                    $secret = bin2hex(openssl_random_pseudo_bytes(4));
```

Comment:
@enishant , can we use better password generation with at least numbers, Capital and small case letters.

Reply:
@ramth05 Added password generation with at numbers, Capital and small case letters.